package.spec

```diff
@@ -0,0 +1,41 @@
+%define pkg_name tendrl_frontend
+%define pkg_version 0.0.1
+%define pkg_release 1
+
```

Comment:
Use package name, version, release directly like
Name: tendrl-frontend
version: 0.0.1
release: 1%{?dist}

and use %{name}